dissect/target/plugin.py

```diff
@@ -844,6 +978,9 @@ def all_plugins():
         if "get_all_records" in available["exports"]:
             available["exports"].remove("get_all_records")
         modulepath = available["module"]
+        # Always skip these
+        if available["class"] in ["DefaultPlugin", "ExamplePlugin"]:
```

Comment:
I don't really like this being a string based skip, is there nowhere else this could be more accurately checked?